cluster_traffic_capture/docker_config/docker-entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+#
+# This file take from here [1], without modification.  Not sure exactly what it does yet.
```

Comment:
not exactly sure what it does yet?
Isn't this the entry point?

Reply:
Sort of.  The distinction between ENTRYPOINT and CMD in a Dockerfile is a bit nuanced; I'm unclear precisely how this is used.  On top of that, I haven't fully grok'd what this bash code is doing.  That said... it seems to "work", or, at least, doesn't break anything.